removed alpha chain oracle for devrel node, with ap election data (#456)
smartcontractkit/documentation: docs/Using Any API/existing-job-request.md

  image:
    0: "/files/OpenGraph_V3.png"
---
Using an *existing* Oracle Job makes your smart contract code more succinct. This page explains how to retrieve the winners of various elections based on the date, state, and race ID.

# AP Election Data Consumer

In [Make a GET Request](../make-a-http-get-request/), the example contract code declared which URL to use, where to find the data in the response, and how to convert it so that it can be represented on-chain.

This example uses an existing job that is pre-configured to make requests to get [AP Election data](https://developer.ap.org/ap-elections-api/). 

> > 📘 Job Spec
> > The job spec for the Chainlink node in this example can be [found here](https://github.com/smartcontractkit/documentation/_includes/samples/APIRequests/ap-election-job.md).

Reviewer: ```suggestion
You can find the job spec for this example on [GitHub](https://github.com/smartcontractkit/documentation/tree/main/_includes/samples/APIRequests/ap-election-job.toml).
```
